server.js

```diff
@@ -68,13 +65,14 @@ app.get('/api/poll', (request, response) => {
 });
 
 app.post('/api/poll', (request, response) => {
+	response.setHeader("Content-Type", "application/json");
```

Comment:
I set a header that corresponds to the ajax call, hoping that would fix the data type conflict, but it did not.